### PR TITLE
feat(@vtmn/react): add select component

### PR DESCRIPTION
**Pull request by Thibault-DECAT**
## Pull Request checklist

🚨 Please review the [guidelines for contributing](../blob/main/CONTRIBUTING.md) to this repository.

- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch**. Don't request your main!
- [x] Check the commit's or even all commits' message styles matches our requested structure.
- [x] Check your code additions will fail neither code linting checks nor unit test.
- [x] Check PR name, it must follow the https://www.conventionalcommits.org pattern
- [x] If it's a new component in CSS, ask for design review

## Does this introduce a breaking change?

- No

### CODE REVIEW

File: packages/sources/react/src/components/VtmnSelect/VtmnSelect.tsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+      )}
+    </div>
+  );
+};
+
+const MemoVtmnBadge = React.memo(VtmnSelect);
+
+MemoVtmnBadge.displayName = 'VtmnSelect';
+
+export default MemoVtmnBadge;
```
Comment (lauthieb) [inline packages/sources/react/src/components/VtmnSelect/VtmnSelect.tsx:58]:
```suggestion
const MemoVtmnSelect = React.memo(VtmnSelect);

MemoVtmnSelect.displayName = 'VtmnSelect';

export default MemoVtmnSelect;
```

File: packages/sources/react/src/index.ts [hunk 2]
```diff
@@ -11,6 +11,7 @@ export { VtmnRadioButton } from './components/VtmnRadioButton';
 export { VtmnTextInput } from './components/VtmnTextInput';
 export { VtmnToggle } from './components/VtmnToggle';
 export { VtmnTooltip } from './components/VtmnTooltip';
+export { VtmnSelect } from './components/VtmnSelect';
```
Comment (lauthieb) [inline packages/sources/react/src/index.ts:14]: Can you please reorder alphabetically? Thanks!

File: packages/sources/react/src/index.ts [hunk 3]
```diff
@@ -23,3 +24,4 @@ export type { VtmnRadioProps } from './components/VtmnRadioButton';
 export type { VtmnTextInputProps } from './components/VtmnTextInput';
 export type { VtmnToggleProps } from './components/VtmnToggle';
 export type { VtmnTooltipProps } from './components/VtmnTooltip';
+export type { VtmnSelectProps } from './components/VtmnSelect';
```
Comment (lauthieb) [inline packages/sources/react/src/index.ts:27]: Can you also please reorder alphabetically? Thanks!

**Review by lauthieb — CHANGES_REQUESTED** [review]
Thanks a lot @Thibault-DECAT for this contribution. 
Just some few changes :-)

File: packages/sources/react/src/index.ts (hunk 2 above)
Comment (Thibault-DECAT) [reply packages/sources/react/src/index.ts]: done

File: packages/sources/react/src/index.ts (hunk 3 above)
Comment (Thibault-DECAT) [reply packages/sources/react/src/index.ts]: done

File: packages/sources/react/src/components/VtmnSelect/VtmnSelect.tsx [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+        id={id}
+        className={clsx('vtmn-select', className, {
+          'vtmn-select--error': error,
+        })}
+        aria-describedby={hasErrorText ? errorTextId : undefined}
+        {...props}
+      >
+        {options.map((option) => {
+          return option;
+        })}
```
Comment (gabbloquet) [inline packages/sources/react/src/components/VtmnSelect/VtmnSelect.tsx:42]:
```suggestion
        {options.map((option) => option )}
```

File: packages/showcases/core/csf/components/VtmnSelect.csf.js [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+    description: 'The label of the select.',
+    defaultValue: 'Label',
+    control: { type: 'text' },
+  },
+};
+
+export const parameters = {
+  design: {
+    type: 'figma',
+    url: 'https://www.figma.com/file/zDZIyayUlr1yTWrsi7cFoo/?node-id=2756%3A10788',
```
Comment (gabbloquet) [inline packages/showcases/core/csf/components/VtmnSelect.csf.js:37]: That's not the right link I think ? 🤔

Comment (lauthieb) [reply packages/showcases/core/csf/components/VtmnSelect.csf.js:37]: u're right, just proposed a new one :)

Comment (lauthieb) [inline packages/showcases/core/csf/components/VtmnSelect.csf.js:37]:
```suggestion
    url: 'https://www.figma.com/file/zDZIyayUlr1yTWrsi7cFoo/?node-id=2535%3A52',
```

**Review by lauthieb — DISMISSED** [review]

**Comment by lauthieb** [comment]
Thanks a lot @Thibault-DECAT pour your contribution!

**Pull request merged**